assets/scripts/admin/questionEditor.js

```diff
@@ -95,8 +95,8 @@ $(document).on('ready pjax:scriptcomplete', function () {
    */
   function updateRowProperties() {
     var sID = $('input[name=sid]').val();
-    var gID = $('input[name=gid]').val();
-    var qID = $('input[name=qid]').val();
+    var gID = $('[name=question\\[gid\\]]').val();
```

Comment:
Maybe 

```
var qID = false;
if($('[name=question\\[gid\\]]').length) {
    var qID =$('[name=question\\[gid\\]]').val();
}
```

Else : no way to set it with a better solution ? Why not use data-qid or data-gid for example ?